docker/docker/src/main/java/org/arquillian/cube/docker/impl/util/AutoStartOrderUtil.java

```diff
@@ -70,6 +86,14 @@
         return new HashSet<>(nodes.values());
     }
 
+    private static boolean isRegularExpressionBased(String[] autoStartContainers) {
+        return autoStartContainers.length == 1;
+    }
+
+    private static boolean isCommaSeparatedBased(String[] autoStartContainers) {
+        return autoStartContainers.length > 1;
```

Comment:
I thought about this case but I think that it would be quite strange to give a container a name with ',' maybe not but a bit strange